source/types/options.ts

```diff
@@ -2,6 +2,8 @@ import type {LiteralUnion, Required} from './common.js';
 import type {Hooks} from './hooks.js';
 import type {RetryOptions} from './retry.js';
 
+export type SearchParamsInit = string | string[][] | Record<string, string> | URLSearchParams | undefined;
```

Comment:
Isn't this type already available in the DOM lib?

Reply:
Haven't found it: https://github.com/microsoft/TypeScript/blob/master/lib/lib.dom.d.ts#L16108